src/GitVersionCore/AssemblyInfo.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System.Reflection;
+using System.Reflection;
 using System.Runtime.CompilerServices;
 
 [assembly: AssemblyTitle("GitVersionCore")]
 [assembly: AssemblyProduct("GitVersion")]
-[assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.0.0")]
+[assembly: AssemblyVersion("4.0.0.0")]
```

Comment:
Isn't this set through GitVersion in the build process and should be kept to 1.0.0 in the repo?